.github/workflows/cypress.yml

```diff
@@ -6,6 +6,7 @@ jobs:
     runs-on: ubuntu-20.04
     env:
       MAAS_URL: http://localhost:5240
+      MAAS_DOMAIN: localhost
```

Comment:
all the places you use this is where you could use MAAS_URL and avoid repeating `http://` ... `:5240`

Reply:
You are correct. There's a reason why I marked this PR as a draft 😄